src/main/scala/inox/ast/SymbolOps.scala

```diff
@@ -1053,6 +1054,7 @@ trait SymbolOps { self: TypeOps =>
             nl
 
           case fi: FunctionInvocation =>
+            // FIXME should this also use "open" bounds?
```

Comment:
No, I don't think so. We are only interested in let bindings where the body (transitively) contains a function invocation, which obviously isn't possible for an open bound.